megamek/src/megamek/client/ui/swing/MegaMekGUI.java

```diff
@@ -292,29 +292,36 @@ private void showMainMenu() {
                 }
             }
         } else {
-            splashFilename = FILENAME_MEGAMEK_SPLASH;
             backgroundIcon = null;
         }
-        File splashFile = new File(Configuration.widgetsDir(), splashFilename);
-        // Ensure we have a splash image, even if we have to fall back to default.
-        if (!splashFile.exists()) {
-            splashFilename = FILENAME_MEGAMEK_SPLASH;
-        }
-        // initialize splash image
-        Image imgSplash = frame.getToolkit()
-                .getImage(new MegaMekFile(Configuration.widgetsDir(), splashFilename).toString());
 
-        // wait for splash image to load completely
-        MediaTracker tracker = new MediaTracker(frame);
-        tracker.addImage(imgSplash, 0);
-        try {
-            tracker.waitForID(0);
-        } catch (InterruptedException e) {
-            // really should never come here
+        // Use the current monitor so we don't "overflow" computers whose primary
+        // displays aren't as large as their secondary displays.
+        DisplayMode currentMonitor = frame.getGraphicsConfiguration().getDevice().getDisplayMode();
+        int monitorW = currentMonitor.getWidth();
+        int monitorH = currentMonitor.getHeight();
+
+        int pixelPerInch= java.awt.Toolkit.getDefaultToolkit().getScreenResolution();
```

Comment:
Just using Toolkit instead of the full dependency path is fine. We allow wildcard awt and swing imports.